db/migrate/20221017202611_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true, index: true
```

Comment:
- [ ] According to the project requirement kindly change the reference to user to author i.e `  t.references :author, foreign_key: { to_table: 'users' }`